akka-persistence/src/main/resources/reference.conf

```diff
@@ -41,6 +41,12 @@ akka.persistence {
     no-snapshot-store {
       class = "akka.persistence.snapshot.NoSnapshotStore"
     }
+    recovery {
+      # There is more time in between individual events gotten from the journal
+      # during replay than this the replay will be failed and behave which causes
+      # the actor to stop (after onRecoveryFailure has been invoked)
+      event-timeout = 30s
+    }
```

Comment:
I don't think we need the nesting. We already have other properties that would have gone into that section if we had it from the beginning, but that's too late now. `recovery-event-timeout`